contracts/cw4-stake/src/contract.rs

```diff
@@ -213,9 +228,17 @@ pub fn handle_claim(
     }
 
     let config = CONFIG.load(deps.storage)?;
-    let amount = coins(release.u128(), config.denom);
+    let amount;
+    if let Denom::Native(denom) = &config.denom {
```

Comment:
You can use a match here, then you don't need the final else (match can prove it covers all enum variants)